package.json

```diff
@@ -7,9 +7,10 @@
   "main": "src/index.jsx",
   "dependencies": {
     "@kbss-cvut/s-forms": "^0.7.2",
+    "@mui/material": "^6.1.2",
```

Comment:
I do not recommend installing MUI if we already use Bootstrap. We should use one or the other, but the idea behind these toolkits is to keep consistency across the app.

Reply:
ok, I replaced it with the bootstrap classes, looks similar to me
![bootstrap_dashboard](https://github.com/user-attachments/assets/68764301-24cc-4c83-a663-0992a888dbd4)

Reply:
@LaChope 

Could you help me with the gaps between elements in a row for a large screen in the [dashboard](https://github.com/kbss-cvut/record-manager-ui/blob/43fa980f944e6863cdeccd45ee87398dff3f0cf1/src/components/dashboard/Dashboard.jsx)? I tried to debug, but I didn't understand what styles are applied on a large screen.
.